exercises/proverb/Sources/ProverbExample.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+struct Proverb {
+
+    func recite(_ pieces: [String]) -> String {
+
+    var saying = String()
+
+    if let firstPiece = pieces.first {
```

Comment:
You can check if the array is empty instead of checking is it has a first element

Reply:
This was my suggestion. It avoids the need to force unwrap the first element when it's used below

Reply:
sounds good! Usually I do something like `guard !array.isEmpty() else {...}` but it really comes down to style